docs/how-tos/trigger-sync.md

```diff
@@ -1,61 +1,93 @@
 ---
-title: "Trigger Cloud Sync Conditionally"
-linkTitle: "Trigger Sync"
-description: "Trigger cloud sync to sync captured data conditionally."
-weight: 20
+title: "How to trigger cloud sync conditionally"
+linkTitle: "Trigger data sync conditionally"
+description: "Trigger cloud sync to sync captured data when custom conditions are met."
+weight: 44
 type: "docs"
 tags: ["data management", "cloud", "sync"]
+images: ["/services/icons/data-cloud-sync.svg"]
 aliases:
   - /data/trigger-sync/
+  - /services/data/trigger-sync/
 no_service: true
-# SME: Alexa Greenberg
 ---
 
-You can use a {{< glossary_tooltip term_id="module" text="module" >}} to sync data only when a certain logic condition is met, instead of at a regular time interval.
+You may want to sync data only when a certain logic condition is met, instead of at a regular time interval.
 For example, if you rely on mobile data but have intermittent WiFi connection in certain locations or at certain times of the day, you may want to trigger sync to only occur when these conditions are met.
 Or, you may want to trigger sync only when your machine detects an object of a certain color.
 The code for both of these examples is provided by the [trigger-sync-examples module](https://github.com/viam-labs/trigger-sync-examples-v2), and you can [create your own module](/how-tos/create-module/) if you want to use different logic.
 
+To set up conditional syncing you need to:
+
+1. Create or use an existing module that implements a `sensor` component that uses your custom logic to determine when to enable and disable sync.
+   While this sensor is not sensing the time it _senses_ whether the data manager should sync or not.
```

Comment:
I don't understand this? why should it be sensing time?
```suggestion
   This sensor _senses_ whether the data manager should sync or not.
```

Reply:
This is the unchanged text of the original doc. I think this helps clarify because it's a pretty abstract sensor. When a reader reads "time sync sensor" they'd not know what it senses and it could just as easily "sense" time as "sense" what it actually "senses"

Reply:
Would this make more sense?
```suggestion
   This sensor does not sense time.
   It _senses_ whether the data manager should sync or not.
```